governator-core/src/main/java/com/netflix/governator/LifecycleListener.java

```diff
@@ -10,8 +10,6 @@
  * inject all dependencies lazily using {@link Provider} injection.  Otherwise,
  * these dependencies will be instantiated too early thereby bypassing lifecycle 
  * features in LifecycleModule.
- * 
- * @author elandau
  */
 public interface LifecycleListener {
```

Comment:
I found a couple uses of this that will probably break with an extra method present.